deploy/chart/crds/0000_50_olm_00-clusterserviceversions.crd.yaml

```diff
@@ -662,13 +662,19 @@ spec:
                       deployments:
                         type: array
                         items:
-                          description: StrategyDeploymentSpec contains the name and
-                            spec for the deployment ALM should create
+                          description: StrategyDeploymentSpec contains the name, spec
+                            and labels for the deployment ALM should create
                           type: object
                           required:
                           - name
                           - spec
                           properties:
+                            label:
+                              description: Set is a map of label:value. It implements
+                                Labels.
```

Comment:
Isn't this suppose to be a user readable message? i.e It's a help message for someone who's not looking at the code at all, who's possibly looking at this by using `kubectl describe`, and therefore should not contain any `implement`ation details. 
Also not sure why this is being added in this PR?

Reply:
The description here is autogenerated based on [comments](https://github.com/kubernetes/apimachinery/blob/master/pkg/labels/labels.go#L34) above the k8s.io/apimachinery/pkg/labels.Set type.

Reply:
I see. That's unfortunate coz as an user this would mean nothing to me. Will we be breaking the convention  if we override the description here?